server/graphql/v2/object/Host.js

```diff
@@ -130,7 +130,7 @@ export const Host = new GraphQLObjectType({
           if (connectedAccounts?.find?.(c => c.service === 'transferwise')) {
             supportedPayoutMethods.push(PayoutMethodTypes.BANK_ACCOUNT);
           }
-          if (connectedAccounts?.find?.(c => c.service === 'paypal') || !collective.settings?.disablePaypalPayouts) {
+          if (connectedAccounts?.find?.(c => c.service === 'paypal') && !collective.settings?.disablePaypalPayouts) {
```

Comment:
Are you sure about this?

With the old PayPal payouts, a Connected Account was not necessarily needed?

Reply:
Unless 100% sure, I would not touch this now.

Reply:
You're correct, I'll fix this.